packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -157,53 +156,43 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
   }), [state, domRef, direction, collator]);
 
   let provider = useProvider();
-  let {checkboxProps} = useGridSelectionCheckbox({key: null}, state);
   let dragState: DraggableCollectionState;
   if (isListDraggable) {
     dragState = dragHooks.useDraggableCollectionState({
       collection: state.collection,
       selectionManager: state.selectionManager,
-      renderPreview(selectedKeys, draggedKey) {
+      renderPreview(draggingKeys, draggedKey) {
         let item = state.collection.getItem(draggedKey);
-        let itemWidth = domRef.current.offsetWidth;
-        let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
-        let isSelected = state.selectionManager.isSelected(item.key);
+        let isDraggingMultiple = draggingKeys.size > 1;
+
         return (
           <Provider
```

Comment:
Ideally I think we'd move this into a separate component, but ok for now.

Reply:
Went ahead and moved its own component. Was originally running into issues because I calling useProvider from inside and that wasn't working. Ended up passing in provider and direction via props which fixed the issue.